### PR TITLE
added vrv

### PR DESCRIPTION
**Pull request by tejassharma96**

### CODE REVIEW

File: strategies/vrv.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var playerName = 'VRV'
+
+var getTitle = function() {
+  return document.querySelector(".series").innerText
```
Comment (matthewdias) [inline strategies/vrv.js]: can you use single quotes throughout this file?

File: strategies/vrv.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+var getEpisode = function() {
+  return parseInt(document.querySelector('.title').innerText.split(' ').slice(1, 2)[0])
+}
+
+var getPlaying = function() {
+  return !(document.querySelector("#player > video").paused)
```
Comment (matthewdias) [inline strategies/vrv.js]: Is there ever a chance of there being two video elements rendered at the same time? If not we can get rid of the `#player > `.

File: strategies/vrv.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+var playPause = function() {
+  var video = document.querySelector("#player > video")
+  if (video.paused
+)    video.play()
```
Comment (matthewdias) [inline strategies/vrv.js]: move this closing paren to the previous line

**Review by matthewdias — CHANGES_REQUESTED** [review]
Thanks! Just some ocd stuff. Sorry for being anal about this 😅

**Comment by tejassharma96** [comment]
ok, so after further testing, VRV appears to just not want to give me the video player info sometimes... Like sometimes I can access it with just document.querySelector('video'), and sometimes it just consistently gives me nothing. I'm gonna remove the video control stuff completely tbh and add a * to the readme for VRV

**Review by matthewdias — APPROVED** [review]

**Pull request merged**